lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1027,13 +1174,52 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
     assert(isEnabled);
 
     final EditingState newEditingState = EditingState.fromDomElement(activeDomElement);
+    TextEditingDeltaState? newTextEditingDeltaState;
+    if (inputConfiguration.enableDeltaModel) {
+      newTextEditingDeltaState = TextEditingDeltaState.inferDeltaState(newEditingState, lastEditingState, editingDelta);
+    }
 
     if (newEditingState != lastEditingState) {
       lastEditingState = newEditingState;
-      onChange!(lastEditingState);
+      _editingDelta = newTextEditingDeltaState;
+      onChange!(lastEditingState, editingDelta);
```

Comment:
Did you mean to pass editingDelta here and not newTextEditingDeltaState? (Or per my comment above, maybe they are equivalent.)  I'm confused by the naming here in general.  Is it possible to make it more consistent with the naming for the non-delta variables here?  (Where lastEditingState becomes newEditingState and is passed to onChange).

Reply:
At this point newTextEditingDeltaState is equivalent to editingDelta. On line 1184 we set _editingDelta to newTextEditingDeltaState, and then editingDelta that is passed to onChange is equivalent since it is a getter for _editingDelta.

Here I was trying to follow the same pattern the editingState uses. First getting the new editing state from the DOM; I get the new editingDelta from inferDeltaState. And then lastEditingState is set to the newEditingState; I set the _editingDelta to the newTextEditingDeltaState. And then both of these are passed to onChange. 

I normalized some of the names in a recent change. Let me know if it's still confusing.